sections/appendix/stemming.tex

```diff
@@ -1,24 +1,29 @@
 \subsection{Stemming the dataset}\label{sec:stemming}
-\vejleder{Se dokument for flere kommentarer}
-In this project, we have done minimal preprocessing \vejleder{hvad med stop ord, fylde ord?}of the dataset, because, in a previous project, a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
+In this project, we have done minimal preprocessing of the dataset.
+Stop words have not been specifically removed, but most of these are naturally removed since we filter all words that appear in more than $10\%$ of the documents.
+We do minimal preprocessing, because in a previous project a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
 This made us completely avoid trying to include a stemming process, even though this was only a smaller part of this previous preprocessing step.
 Though, after further consideration, it is worth looking into what effect adding just stemming to our current preprocessing, described in \autoref{sec:dataset}, would have on the standard \gls{lda} model.
 
 With stemming included, the number of unique words goes down to $60,651$ from the previous $69,192$ unique words.
 This means that 8541 unique words have been stemmed down to their root forms.
 Even though much fewer unique words are in our dataset, which can have an effect on which articles are removed, the dataset only goes from $139,064$ articles to $139,060$, which is a removal of 4 articles.
+There is most likely fewer articles, because when we filter words that appear in more than $10\%$ of the documents after stemming, some new documents may end up empty.
 
 In the non-stemmed \gls{lda} model, words that have no meaning topically seem to have a large influence.
 This is still the case in the stemmed model, where words like 'du' (you), 'mig' (me), and 'a' still appear in the top words of topics.
 To remove these words, we would have to include some more advanced preprocessing in the form of stop word removal and part-of-speech (POS) tagging.
+From experience, POS taggers can not figure out the semantic meaning of words that are spelled the same, and then we would somehow have to chose which meaning is correct, possibly creating new errors in our data.
 
 Examples of top words for similar topics between the models can be seen in \autoref{tab:topic_examples}.
 It is seen here that some of the topics between the models are clearly about the same subjects.
-The stemmed model's topics might also be slightly more understandable because words with the same meaning (e.g. 'politi' and 'politiet') do not appear.
+This is also generally seen for other topics throughout the two models.
+A positive effect of having done stemming, is that the stemmed model's topics might also be slightly more understandable, because words with the same meaning (e.g. 'politi' and 'politiet') do not appear.
```

Comment:
```suggestion
A positive effect of having done stemming is that the stemmed model's topics might also be slightly more understandable because words with the same meaning (e.g. 'politi' and 'politiet') do not appear.
```